spec/support/flow_helpers.rb

```diff
@@ -157,6 +157,16 @@ def fill_in_vehicle_details_screen(screen_name: :vehicle_details, form_name: :cl
   click_on "Save and continue"
 end
 
+def fill_in_vehicles_details_screen(screen_name: :vehicles_details, vehicle_finance: "0")
+  confirm_screen screen_name
```

Comment:
We don't need `screen_name` to be passed in as an argument - it's always going to be "vehicles_details". The only reason it's an argument in some methods in this file is when we use the same method to fill out both the client and partner versions of a screen, which have different screen names.